zk-token-sdk/src/encryption/auth_encryption.rs

```diff
@@ -82,11 +90,17 @@ impl AuthenticatedEncryption {
 #[derive(Debug, Zeroize)]
 pub struct AeKey([u8; 16]);
 impl AeKey {
+    /// Deterministically derives an authenticated encryption key from a Solana signer and a tag.
```

Comment:
As I'm reading these docs, I'm not loving the word `tag` for the byte slice. Really, `b"AeKey"` seems more like the tag. Is there a better var name we can use?
That change doesn't need to be part of this PR.

Reply:
Hm, we are essentially using it like a seed, but "seed" for a key generally refers to private bytes. What do you think of using `public_seed` instead of `tag`?

Reply:
Yeah, `seed` was where my head went too. I like `public_seed`!